vmu.py

```diff
@@ -33,48 +33,32 @@ def load_run_params(param_dir):
             # skip non-yaml files
             pass
         else:
-            if param_contents.has_key('platform') \
-               and param_contents.has_key('sources') \
-               and param_contents.has_key('packages'):
+            if set(param_contents.keys()) == set(['platforms', 'sources', 'package_sets']):
```

Comment:
In the unimportant department, the best of both worlds is probably `set(param_contents)`.

(When you try to iterate over a dict, you actually iterate over the keys.)